client-grpc/Cargo.toml

```diff
@@ -10,13 +10,35 @@ keywords.workspace     = true
 license-file.workspace = true
 repository.workspace   = true
 
+[features]
+# Will implement mock functions for the client, only use for tests!
+mock_client = ["tower", "tokio", "lib-common/grpc_mock", "svc-template-rust"]
+
 [dependencies]
-prost       = "0.11"
-prost-types = "0.11"
-tonic       = "0.8"
+log               = { version = "0.4" }
+prost             = "0.11"
+prost-types       = "0.11"
+svc-template-rust = { path = "../server", optional = true }
+tonic             = "0.8"
+tower             = { version = "0.4", optional = true }
+
+[dependencies.lib-common]
+features = ["grpc"]
+git      = "https://github.com/Arrow-air/lib-common.git"
+tag      = "v0.1.1-develop.2"
```

Comment:
`develop` to keep it to latest? this tag will be updated no matter what by the devs who are working on a repo cloned from this template so it doesn't matter too much I suppose

Reply:
We currently don't have a way to point to the latest develop code other than using the branch, which results in Cargo issues if the branch gets updated with a new ref.
If anything, we should update this to point to the lates release version once lib-common get's released.